src/main/java/jenkins/plugins/logstash/persistence/BuildData.java

```diff
@@ -178,17 +178,23 @@ public int getPassCount()
   private Map<String, String> buildVariables;
   private Set<String> sensitiveBuildVariables;
   private TestData testResults = null;
-
-  // Freestyle project build
+  
   public BuildData(AbstractBuild<?, ?> build, Date currentTime, TaskListener listener) {
+    this(build, currentTime, listener, null);
+  }
+  // Freestyle project build
+  public BuildData(AbstractBuild<?, ?> build, Date currentTime, TaskListener listener, Map<String, String> additionalParams) {
```

Comment:
it seems there is no way to use this additional params in freestile builds, right?

Reply:
why not?

Reply:
I don't see any stapler bindings updated, how would it be configured?

Reply:
For now it won't be implemented in freestyle builds

Reply:
OK, can you please remove those extra constructors?
Otherwise LGTM